wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -129,7 +129,6 @@ public final class MLSService: MLSServiceInterface {
         case keyPackageQueriedTime
     }
 
-    var backendPublicKeys = BackendMLSPublicKeys()
```

Comment:
There's no point storing this as we fetch it every time we create a group anyway.

Reply:
I think it was to have an optimization, since we shouldn't need to fetch everytime. But we can do that in a second time if still relevant cc @typfel